library/core/src/lib.rs

```diff
@@ -156,7 +156,6 @@
 #![feature(const_slice_from_ref)]
 #![feature(const_slice_index)]
 #![feature(const_is_char_boundary)]
-//
```

Comment:
Unrelated change, probably should undo (the extra space does give some sense of separation between blocks that isn’t really there with just a comment alone)